src/lib.rs

```diff
@@ -433,12 +433,20 @@ impl<'input, Endian> Unit<'input, Endian>
 
             // Extract source file and line information about the compilation unit
             let line_offset = match entry.attr_value(gimli::DW_AT_stmt_list) {
-                Some(gimli::AttributeValue::DebugLineRef(offset)) => offset,
+                Ok(Some(gimli::AttributeValue::DebugLineRef(offset))) => offset,
+                Err(e) => {
+                    return Err(Error::from(ErrorKind::Gimli(e)))
+                        .chain_err(|| "invalid compilation unit statement list")
```

Comment:
Yep, but when I read an error chain that says "invalid compilation unit statement list" I expect that to mean that the statement list is invalid, but it may not be. I'm not sure how to reword it to be concise and accurate. It's a minor point though, and maybe not worth changing, what's there is certainly good enough for debugging.